File: simpleClass/scala/src/start.scala

import model.UserInfo

import scala.collection.mutable.ListBuffer

object  main extends App {

  // Inicia uma lista modificavel
  var list = new ListBuffer[UserInfo]()

  // Obtem o tempo inicial
  val antes = System.currentTimeMillis()

  val size: Int = 1000000

  // Acidionando elementos unicos na lista
  for (index <- 1 to size) {
    val user: String = "user" + index
    val password: String = "password" + index
    list += new UserInfo(user, password)
  }

  // Obtem o tempo final
  val depois = System.currentTimeMillis()

  // Exibe o resultado
  print(depois-antes)
}

File: languages/java/src/box/start.java
package box;

import java.util.ArrayList;
import java.util.List;

/**
 Classe inicial do programa
 */
public class start {

    /**
     * Método de inicialização do projeto
     * @param args lista de parametros passados via console
     */
    public static void main(String args[]){

        // Gera uma lista de UserInfo
        List<UserInfo> list = new ArrayList<>();

        long antes = System.currentTimeMillis();

        for(int index = 0; index < 1000000; index++){
            String user = "user" + index;
            String password = "password" + index;
            list.add(new UserInfo(user, password));
        }

        long depois = System.currentTimeMillis();

        System.out.println(list.get(0).getPassword());
        System.out.println("Diff: " + (depois - antes));
    }
}
